tools/topology/topology2/platform/intel/sdw-jack-generic.conf

```diff
@@ -81,11 +81,50 @@ Object.Pipeline {
 			Object.Widget.host-copier.1 {
 				stream_name	"Passthrough Capture 0"
 				pcm_id 1
+				num_input_audio_formats 9
+				num_output_audio_formats 9
 				Object.Base.audio_format.1 {
 					# 32/32 -> 16/16 bits conversion is done here
 					in_bit_depth	32
 					in_valid_bit_depth	32
 				}
+				# 32-bit 48KHz 1ch
```

Comment:
sorry clicked on approve before I realized this. We should not use the audio_format class anymore. Please split this into input_audio_format and output_audio_format objects please

Reply:
> sorry clicked on approve before I realized this. We should not use the audio_format class anymore. Please split this into input_audio_format and output_audio_format objects please

Done